language/types_values_variables.md

```diff
@@ -269,7 +274,7 @@ Represents the abstract notion of "value", its subtypes are `Numeric`, `String`
 
 ### Numeric
 
-Represents the abstract notion of "number", its subtypes are `Integer`, and `Float`.
+Represents the abstract notion of "number", its subtypes are `Integer`, `Float`, `Timespan` and `Timestamp`.
```

Comment:
Changing this isn't just confined to the spec. It will break the implementation as well. Things like common type between a Float and Timespan for instance. It can no longer be Numeric.